src/telemetry.ts

```diff
@@ -183,6 +183,9 @@ function sendTelemetryEvent(
 	properties?: { [key: string]: string },
 	measures?: { [key: string]: number }
 ): void {
+	if (!aiKey) {
```

Comment:
The `aiKey` is initialized in line 12 and is a constant.
Therefore, in its current state, the extension will not reach a state where this will be empty..

Reply:
https://github.com/golang/vscode-go/commit/ada08587033b59b41f093fc39f8890350d2dbacd#diff-499709399145585d79806a7415935b15L12 is the reasoning behind this.